### PR TITLE
Bug 2052124: add updateStrategy to DefaultCatsrc.yaml

### PR DESCRIPTION
**Pull request by sakhoury**
`DefaultCatsrc.yaml` in ztp source-crs has registry poll disabled, therefore an updated operator index image will not be picked up automatically, thus operator upgrade will not take place.

This commit adds an `updateStrategy` specification to the `DefaultCatsrc.yaml` with a registry polling interval of `1h`.

The polling interval was determined through an extensive empirical investigation. The report is attached to the PR: [polling interval characterization.pdf](https://github.com/openshift-kni/cnf-features-deploy/files/8812345/polling.interval.characterization.pdf)

Signed-off-by: Sharat Akhoury <sakhoury@redhat.com>

/cc @lack @imiller0 @tliu2021 @Missxiaoguo

### CODE REVIEW

File: ztp/source-crs/DefaultCatsrc.yaml [hunk 1]
```diff
@@ -11,3 +11,6 @@ spec:
     image: $imageUrl
     publisher: Red Hat
     sourceType: grpc
+    updateStrategy:
+        registryPoll:
+            interval: 1h
```
Comment (imiller0) [inline ztp/source-crs/DefaultCatsrc.yaml:16]: Please ensure there is a newline at the end of the file to avoid github flagging this.

Comment (sakhoury) [reply ztp/source-crs/DefaultCatsrc.yaml]: Done

**Comment by sakhoury** [comment]
Closing this PR. Will re-create with a new one.

**Pull request closed without merging**